Makefile

```diff
@@ -15,8 +15,12 @@ deps:
 	go get -u $(BUILDFLAGS) github.com/golang/lint/golint
 	go get $(BUILDFLAGS) github.com/vbatts/git-validation
 
-test:
-	@go test $(BUILDFLAGS) -cover ./...
+vndr:
+	go get -u github.com/LK4D4/vndr
+	vndr
+
+test: vndr
```

Comment:
The original goal of `test-skopeo` was to stop us from constantly breaking `test-skopeo` with API changes, without even realizing that we broke it; I think that still very much applies.

(In principle, we could instead have some kind of public API list and just fail tests every time we break the API. But at least the two of us would end up using that test failure to update skopeo, so for us it amounts to the same thing; not sure about other contributors.)